wicket-jquery-ui-samples/src/main/java/com/googlecode/wicket/jquery/ui/samples/pages/kendo/menu/KendoContextMenuPage.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html xmlns:wicket="http://wicket.apache.org">
+<head>
+<wicket:head>
+	<title>Wicket - jQuery UI: kendo-ui context menu</title>
+</wicket:head>
+</head>
+<body>
+<wicket:extend>
+	<div wicket:id="menu"></div>
```

Comment:
- [ ] @sebfz1 - Add text like: "right-click anywhere to display the context menu..."